react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.topics = [
```

Comment:
the topics are supposed to be passed into the component on props

Reply:
Topics array was updated to be passed into the component on props.